mmil.raw.html

```diff
@@ -7764,8 +7764,8 @@
 <TR>
   <TD>df-prod and theorems using it</TD>
   <TD><I>none</I></TD>
-  <TD>To define this, will need to tackle all the issues
-  with ~ df-isum plus some more around, for example, not equal to zero
+  <TD>To define this, will need to tackle all the issues analogous
+  to ~ df-isum plus some more around, for example, not equal to zero
```

Comment:
I know I'm a bit formal, but: "we will" / "one will" ?
A few lines above there is "to replaced" --> "to be replaced"

Reply:
Those seem like improvements, will do.